get_dependencies.sh

```diff
@@ -13,5 +13,7 @@ mkdir build && cd build
 cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release ..
 make
 sudo make install
+
+sudo apt-get install libxml2-dev
```

Comment:
Since we have this as a dependency, do we need to also pull it in in the `external` folder?

Reply:
No, we would not need to do that. Just the header files which we have in /external. The .so package will be pulled with the above command.